scenario/src/ErrorReporter.ts

```diff
@@ -18,6 +21,10 @@ class NoErrorReporterType implements ErrorReporter {
   getDetail(error: any, detail: number): string {
     return detail.toString();
   }
+
+  getEncodedCustomError(errorName: string, args: unknown[]): string | null {
+    return null
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/cloudwalk/compound-protocol/pullRequest?prid=9803552)
```suggestion
    return null;
```